RepoCleanup/Functions/CreateOrgWithTeams.cs

```diff
@@ -0,0 +1,152 @@
+            teams.Add(CreateTeamInfo("Devs", "All application developers", true, Permission.write));
+
+            Console.WriteLine("Creating teams for org. Please wait.");
+            foreach (CreateTeamOption team in teams)
+            {
+                response = await GiteaService.CreateTeam(org.Username, team);
+
+                if (response.Success)
+                {
+                    logBuilder.AppendLine($"{DateTime.Now} - Information - Team {team.Name} was successfully added to org: {org.Username}.");
+                }
+                else
+                {
+                    createdTeams = false;
+                    logBuilder.AppendLine($"{DateTime.Now} - Error - Team {team.Name} was not added to org {org.Username}.");
+                    logBuilder.AppendLine($"{DateTime.Now} - Error -  {response.StatusCode}:{JsonSerializer.Serialize(response.ResponseMessage, Globals.SerializerOptions)}.");
+                }
+            }
+
+            if (createdTeams)
+            {
+                Console.WriteLine($"All teams for {org.Username} successfully created.");
+            }
+            else
+            {
+                Console.WriteLine($"Create teams for {org.Username} failed. See log {logName}.");
+            }
+        }
+
+        private static Organisation CollectOrgInfo()
+        {
+            bool isValid = false;
+            string username = string.Empty;
+
+            while (!isValid)
+            {
+                Console.Write("\r\nSet username (shortname) for org: ");
+                username = Console.ReadLine();
+
+                isValid = Regex.IsMatch(username, "^[a-z]+$");
+                if (!isValid)
+                {
+                    Console.WriteLine("Invalid name. Letters a-z are permitted.");
+                }
+            }
+
+            string fullname = string.Empty;
+
+            Console.Write("\r\nSet fullname for org: ");
+            fullname = Console.ReadLine();
+
+            isValid = false;
+            string website = string.Empty;
+
+            while (!isValid)
+            {
+                Console.Write("\r\nSet website for org: ");
+                website = Console.ReadLine();
+
+                isValid = Regex.IsMatch(website, "^[a-zA-Z0-9\\-._/:]*$");
+                if (!isValid)
+                {
+                    Console.WriteLine("Invalid website adress. Letters a-z and characters:'-', '_', '.', '/', ':' are permitted.");
+                }
+            }
+
+            Organisation org = new Organisation();
+            org.Username = username;
+            org.Fullname = fullname;
+            org.Website = website;
+            org.Visibility = "public";
+            org.RepoAdminChangeTeamAccess = false;
+            return org;
+        }
+
+        private static CreateTeamOption CreateTeamInfo(string name, string description, bool canCreateOrgRepo, Permission permission)
```

Comment:
Maybe this could be extracted to a helperclass and used both for this function and [here](https://github.com/Altinn/altinn-tools/blob/205aa36327c364c736f6e9bce1210dc645f2acb8/RepoCleanup/Functions/CreateTeamForOrgs.cs#L153)
Not many lines, but then we could avoid the duplication.

Reply:
Done

Reply:
Done